src/help/getRenderProperties.js

```diff
@@ -52,12 +52,18 @@ function getRenderProperties(element){
 		};
 	}
 	// If canvas (in node)
-	else if(element.getContext){
+	else if(element && element.getContext){
 		return {
 			element: element,
 			renderer: renderers.CanvasRenderer
 		};
 	}
+	else if(element && typeof element === 'object') {
```

Comment:
If you can think of a better way to identify an object render I'm all ears. This is a little weird because technically almost anything that is passed in will be caught by this because most things are objects in javascript.